Eask

```diff
@@ -1,5 +1,5 @@
 (package "lsp-haskell"
-         "1.0"
+         "1.1"
```

Comment:
I considered 1.0.1, but requiring haskell-mode feels like a significant change so 1.1 or 2.0 is probably more appropriate, depending on the versioning policy in use.

Reply:
Make sense, but let's wait what other people think! Thanks!